src/diff/index.js

```diff
@@ -309,8 +303,8 @@ function diffElementNodes(
 	isHydrating
 ) {
 	let i;
-	let oldProps = oldVNode.props;
-	let newProps = newVNode.props;
+	let oldProps = oldVNode.vnode && oldVNode.vnode.props;
```

Comment:
This should become _vnode

Reply:
This doesn't seem possible due to `component._vnode` overriding this to a property that we already have 😅